src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -228,6 +229,7 @@ public void autonomousInit() {
 
     @Override
     public void teleopInit() {
+        controls = new Controls(subsystems);
         if (subsystems.intakeSubsystem != null) {
```

Comment:
yeah this also doesnt work when we disable teleop and enable auto again
we can do CommandScheduler.resetScheduler or smthing
otherwise it should work